### ISSUE TITLE
DataTables - selected rows are not distinguished

### ISSUE THREAD

**Issue by Koojav** [almasaeed2010/AdminLTE#1259]
Hi,
as stated in the title - when using AdminLTE and DataTables with selectable rows the "tr.selected" style is properly applied in the code but no result is visible. 

I've tried `table-hover` and `table-striped` styles for my table - none of them work properly, although `table-hover` does display kind of selection on hover which should also be visible when a row is selected.

**Comment by REJack** [comment]
Check [this example](https://datatables.net/examples/api/select_single_row.html) from DataTables.

**Comment by Koojav** [comment]
@REJack I have checked the examples and the documentation from DataTables website prior to posting.

As previously stated: "tr.selected" style is properly applied in the code but no result is visible.
Appropriate style definition is missing/broken.

I've fixed problem temporarily by adding another style, except the standard one:
`tr.toggleClass('selected');
tr.toggleClass('bg-warning');`

Since it's quite late (01:00) I'll probably remove additional style assignment and redefine appropriate `tr.selected` style in the morning but it would be nice to have that working as expected right out of the box.
